src/types.d.ts

```diff
@@ -255,3 +255,53 @@ export interface Resources {
 export interface MultiAddressByNode {
   [key: string]: string;
 }
+
+// Config interfaces
+interface NodesConfig {
+  name: string;
+  wsPort: number;
+  port: number;
+  flags?: [strings];
+}
+
+interface RelayChainConfig {
+  bin?: string;
+  chain: string;
+  nodes: [NodesConfig];
+  genesis: {
+    runtime: {
+      runtime_genesis_config: {
+        configuration: {
+          config: {
+            validation_upgrade_frequency: number;
+            validation_upgrade_delay: number;
+          };
+        };
+      };
+    };
+  };
+}
+
+interface ParaChainConfig {
```

Comment:
We have a similar intergace `ParachainConfig` we should use a prefix like `PL_` or maybe move those to a diff types.d file.